.gitignore

```diff
@@ -7,3 +7,4 @@ dist
 .vscode
 
 assets/i18n/translate.*.json
+/.idea/
```

Comment:
Would you please remove this unrelated change?

Reply:
sure done.